src/content/upgrades/sharding/index.md

```diff
@@ -92,25 +86,9 @@ This is still an active discussion point. We’ll update these pages once we kno
 
 The Ethereum upgrades are all somewhat interrelated. So let’s recap how the shard chains relate the other upgrades.
 
-### Shards and the beacon chain {#shards-and-beacon-chain}
-
-The Beacon Chain contains all the logic for keeping shards secure and synced up. The Beacon Chain will coordinate the stakers in the network, assigning them to shards they need to work on. And it will also facilitate communication between shards by receiving and storing shard transaction data that is accessible by other shards. This will give shards a snapshot of Ethereum’s state to keep everything up-to-date.
-
-<ButtonLink to="/upgrades/beacon-chain/">
-  The Beacon Chain
-</ButtonLink>
-
-### Shards and The Merge {#shards-and-docking}
-
-By the time additional shards are added, Ethereum Mainnet will already be secured by the Beacon Chain using proof-of-stake. This enables a fertile Mainnet to build shard chains off of, powered by layer 2 solutions that supercharge the scalability.
-
-It remains to be seen whether Mainnet will exist as the only “smart” shard that can handle code execution – but either way, the decision to expand shards can be revisited as needed.
-
-<ButtonLink to="/upgrades/merge/">
-  The Merge
-</ButtonLink>
+### Shards and the Ethereum blockchain {#shards-and-blockchain}
```

Comment:
This section is labeled "Relationship between upgrades" and serves the purpose of directly comparing and tying together the different upgrades. This kinda slims it down by removing the other direct comparisons...

Given the existing content is so out of date, not against stripping this out right now and we can come back to it... obviously this page needs more work.